eclipse_plugin/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry exported="true" kind="lib" path="lib/guava-19.0.jar"/>
```

Comment:
Had some fun figuring what _orbit_ you're talking about:

- [ ] [maven orbit](https://www.google.de/search?q=maven+orbit&espv=2&biw=1920&bih=1115&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjH-Ofx5pPNAhVH1SwKHbdjA2AQ_AUIBigB)
- [ ] [tycho orbit](https://www.google.de/search?q=tycho+orbit&espv=2&biw=1920&bih=1115&source=lnms&tbm=isch&sa=X&ved=0ahUKEwj5qI2u5pPNAhVMiywKHX8OA4IQ_AUIBigB)
- [x] [Eclipse orbit](http://www.eclipse.org/orbit)